tests-score

```diff
@@ -36,13 +36,17 @@ BOTS = os.path.dirname(os.path.realpath(__file__))
 # https://github.com/cockpit-project/cockpituous/blob/master/learn/README.md
 
 # Here we're looking for a field in a record that only has one value
+
```

Comment:
here it would make more sense to put those two newlines above the function description.

Reply:
Well spotted, thanks! Indeed autopep doesn't handle these well. I went through the entire diff and fixed all of these.